frontends/qt/server/server.go

```diff
@@ -121,6 +130,8 @@ func serve(
 	log := logging.Get().WithGroup("server")
 	log.WithField("args", os.Args).Info("Started Qt application")
 	testnet := flag.Bool("testnet", false, "activate testnets")
+	simulatorPort := flag.Int("simulatorPort", 15423, "port for the BitBox02 simulator")
+	useSimulator := flag.Bool("simulator", false, "use the BitBox02 simulator")
```

Comment:
mention that it implies testnet (if you make this change), or mention that testnet is required if you prefer to keep the testnet check